src/Sulu/Bundle/PageBundle/Tests/Unit/Admin/PageAdminTest.php

```diff
@@ -124,9 +124,29 @@ public function testGetConfigWithVersioning()
             true
         );
 
+        $webspace1 = $this->prophesize(Webspace::class);
+        $webspace1->getName()->willReturn('second webspace');
+
+        $webspace2 = $this->prophesize(Webspace::class);
+        $webspace2->getName()->willReturn('first webspace');
```

Comment:
I know that this is mixed in this test file, but I think I would prefer to use the real `Webspace` implementation here. It is the exact lines of code (even less characters), and we could be absolutely sure, that this is working with the real implementation, not only with the mock:

```php
$webspace2 = new Webspace();
$webspace2->setName('first webspace');
```

Reply:
👍